README.md

```diff
@@ -75,6 +75,20 @@ To receive a user by name:
 GET 'https://login.turing.io/api/v1/users/by_name?q=[NAME]'
 ```
 
+To receive a user by ID:
+```
+GET 'https://login.turing.io/api/v1/users/:id'
```

Comment:
It might be good to add an example of the return value.  Also, should we mention that you need to pass along an auth key?